### PR TITLE
feat: add Rancher Academy Longhorn Basis link

### PR DESCRIPTION
**Pull request by derekbit**
#### Which issue(s) this PR fixes:

Issue #

### CODE REVIEW

**Comment by jillian-maroket** [comment]
@derekbit I'm waiting for Alex to comment on the wording.

**Comment by innobead** [comment]
@derekbit lets add Alex and Forbes having review permission here.

**Comment by derekbit** [comment]
> @derekbit lets add Alex and Forbes having review permission here.

Just invited her to the group

image

**Review by robertsirc — COMMENTED** [review]
LGTM

**Review by innobead — DISMISSED** [review]

**Review by jhkrug — APPROVED** [review]
lgtm

File: layouts/partials/home/project-info.html [hunk 1]
```diff
@@ -1,5 +1,25 @@
 <section class="section">
   <div class="container has-text-centered box push-down">
+    <div class="columns is-vcentered is-variable is-20">
+      <div class="column">
+        <p class="title is-size-4 is-size-5-mobile">
+          Sign up for the latest "Longhorn Basics" Online Training.
```
Comment (asettle) [inline layouts/partials/home/project-info.html:6]: Imho doesn't need the inverted commas, it isn't implying anything. It is Longhorn Basics Online training. Maybe italics? @jillian-maroket thoughts?

**Review by asettle — COMMENTED** [review]
Can't approve but consider this a seal of approval!

**Review by jillian-maroket — COMMENTED** [review]
I included my wording suggestions in my reply to Alex.

File: layouts/partials/home/project-info.html (hunk 1 above)
Comment (asettle) [reply layouts/partials/home/project-info.html:6]: Agreed, @derekbit could you make these changes, please?

Comment (derekbit) [reply layouts/partials/home/project-info.html:6]: Sure

**Review by jillian-maroket — DISMISSED** [review]
LGTM. Thanks!

**Review by jillian-maroket — DISMISSED** [review]

**Review by jillian-maroket — APPROVED** [review]

**Pull request merged**